examples/with-cookie-auth/README.md

```diff
@@ -21,13 +22,33 @@ curl https://codeload.github.com/zeit/next.js/tar.gz/canary | tar -xz --strip=2
 cd with-cookie-auth
 ```
 
-Install it and run:
+### Run locally
+
+The repository is setup as a [monorepo](https://zeit.co/examples/monorepo/) so you can deploy it easily running `now` inside the project folder. However, you can't run it the same way locally (yet).
+
+These files make it easier to run the application locally and aren't needed for production:
```

Comment:
Maybe good to add a caveat here "aren't needed for production if you are using Now to deploy".